src/parser/grammar.ne

```diff
@@ -106,19 +106,19 @@ set_operation -> %RESERVED_SET_OPERATION expression:* {%
   })
 %}
 
-expression -> ( simple_expression | asterisk | comma ) {% unwrap %}
+expression -> ( simple_expression | between_predicate | asterisk | comma ) {% unwrap %}
```

Comment:
this seems a little awkward for now but I suppose it'll look better as we expand the grammar more

Reply:
Yeah. This part is not the prettiest for now :P